src/test/ui/async-await/track-caller/issue-104588-no-op-track-caller.stderr

```diff
@@ -0,0 +1,12 @@
+warning: `#[track_caller]` on async functions is a no-op, unless the `closure_track_caller` feature is enabled
+  --> $DIR/issue-104588-no-op-track-caller.rs:4:16
+   |
+LL |   #[track_caller]
+   |  ________________^
+LL | | async fn foo() {}
+   | |_
```

Comment:
Not a huge fan of how the warning looks, for example the ascii arrow points to nothing (there's no text on this line). If someone has a suggestion on how to make this look cleaner overall, please let me know :)

Reply:
You also managed to find a bug in the output here! it should have been `|^` instead of `|_` 🤔

Reply:
oh! interesting haha. yeah i was like "this look weird..."

Reply:
I'll file a bug for this...

Reply:
Filed #105430